blocks/library/freeform/old-editor.js

```diff
@@ -159,7 +165,16 @@ export default class OldEditor extends Component {
 	render() {
 		const { id } = this.props;
 
+		const inspectorControls = (
```

Comment:
Minor: Personal preference, but I don't think the variable assignment adds clarity to justify separating the element from being inlined into the return array.

Reply:
@aduth I've removed the unnecessary var for clarity as suggested